enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/catchup/storecopy/GetIndexSnapshotRequestHandler.java

```diff
@@ -89,13 +88,13 @@ else if ( !isTransactionWithinReach( snapshotRequest.requiredTransactionId(), ch
                     int recordSize = storeFileMetadata.recordSize();
                     storeFileStreamingProtocol.stream( ctx, new StoreResource( file, relativePath, recordSize, pageCache, fs ) );
                 }
-                status = StoreCopyFinishedResponse.Status.SUCCESS;
-            }
-            finally
-            {
-                storeFileStreamingProtocol.end( ctx, status ).addListener( closeablesListener );
```

Comment:
What happens with this closeablesListener stuff?

Reply:
We only used it for the resource iterator in this class and we don't use anything that need to be closed on complete as we do in the PrepareStoreCopyRequestHandler. The resource iterator is not created in a try block instead.